app/views/fields/belongs_to/_form.html.erb

```diff
@@ -17,9 +17,6 @@ that displays all possible records to associate with.
 %>
 
 <%= f.label field.permitted_attribute %>
-<%= f.collection_select(
-  field.permitted_attribute,
-  field.candidate_records,
-  :id,
-  :to_s,
-) %>
+<%= f.select(field.permitted_attribute) do %>
+  <%= options_for_select(field.candidate_options) %>
```

Comment:
I feel like the word `candidate` is redundant here, given that you already have `option` in the same word. In other words, something is either a `candidate` or an `option`, but being both looks strange.

What about `display_associated_resources`?